web/src/frontend/src/nav-soknad/faktum/SelectFaktum.tsx

```diff
@@ -40,7 +40,7 @@ class FaktumSelect extends React.Component<
 				value={getFaktumVerdi(fakta, faktumKey)}
 				bredde={bredde}
 				onChange={(evt: any) =>
-					dispatch(setFaktumVerdi(faktumKey, evt.target.value))}
+					dispatch(setFaktumVerdi(faktumKey, evt.target.value, this.props.fakta))}
```

Comment:
forslag - når man connecter komponenten så kan vi allerede der slå opp faktumet i stedet for at faktaen blir brukt. Da slipper vi å sende fakta rundt overalt og måtte slå opp flere ganger.

Reply:
Virket ikke. Erstattet med finnFaktum() metode.